layouts/_default/list.html

```diff
@@ -1,4 +1,6 @@
 {{- define "main" -}}
-  {{- $list_page := .Site.GetPage "page" (printf "%s%s" .CurrentSection.Dir "_index") -}}
-  {{- partial "helpers/fragments.html" (dict "root" $ "real_page" . "page" $list_page "page_scratch" $.Scratch) -}}
+  {{- if .CurrentSection -}}
```

Comment:
Having a small explanation on what happens could be helpful for other devs down the line.
Why check for currentSection and using _index for getting the list_page etc?
Also what's the difference between real_page and list_page? Someone with less Hugo knowledge might easily be lost here.